src/main/java/bartworks/common/tileentities/multis/MTEDeepEarthHeatingPump.java

```diff
@@ -297,7 +310,7 @@ private boolean consumeFluid(Fluid fluid, long ammount) {
     @Override
     protected void setElectricityStats() {
         try {
-            this.mEUt = this.isPickingPipes ? -60 : -1 * Math.toIntExact(GTValues.V[this.getMinTier()]);
+            this.mEUt = this.isPickingPipes ? -60 : (-480);
```

Comment:
```suggestion
            this.mEUt = this.isPickingPipes ? -60 : (-TierEU.RECIPE_HV);
```
Requires the import commented above